test_chunk_utils.py

```
@@ -59,9 +59,8 @@ def test_create_large_chunks(self):
         self.assertGreater(len(chunks), 1)
 
         # All chunks should have the same heading in metadata
-        for chunk in chunks:
-            self.assertEqual(chunk.metadata["heading"], "# Large Section")
-            self.assertEqual(chunk.source_url, self.test_url)
+        self.assertTrue(all(chunk.metadata["heading"] == "# Large Section" for chunk in chunks))
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)